Pivotal importer fixes

mig5 opened the pull request:
Fixes https://github.com/taigaio/taiga-back/issues/175

 * Set a dummy person in comment when it is undefined.
 * Fix 'scrum or scrum' helper info on jira and pivotal commands.
 * Adds ability to import the Pull Request attributes from Pivotal into Taiga as a Custom Field.
 * Adds ability to capture the 'blocked' story/epic IDs from Pivotal in the 'blocked_text' field in Taiga.

mig5 committed:
Pivotal importer: Set a dummy person in comment when it is undefined.

Fix 'scrum or scrum' helper info on jira and pivotal commands.

mig5 committed:
Convert custom 'Type' field in Pivotal importer to be a dropdown. Add support for Priority custom field. Fix saving multiple custom fields

mig5 committed:
rewrite the Pivotal '--ask-for-users' option to '--map-users' and optimistically try to map to existing users, no prompting

mig5 committed:
Pivotal importer: ensure stories that have multiple owners get assigned as multiple assignees in Taiga

CarlosLVar commented:
Hi, @mig5,

Sorry for the long time to get back to you here. Could you rebase and re-run the tests?

Best!

mig5 committed:
Add support for importing Pull Requests, Blocker text from Pivotal attributes

mig5 commented:
@CarlosLVar I've rebased from latest main. I also added some extra things we had done since for our Pivotal to Taiga migration, such as importing Pull Request links as a custom field, and also capturing the 'Blocked' story IDs as texts for the Taiga blocker field.

I can't run the tests as it appears to require approval first (because it's my first contribution and it's a fork), but you are welcome to run them or fork a copy of my branch to the upstream repo if you would like.

Thanks!
